pyxem/data/_data.py

```diff
@@ -0,0 +1,311 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016-2023 The pyXem developers
+#
+# This file is part of pyXem.
+#
+# pyXem is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# pyXem is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with pyXem.  If not, see <http://www.gnu.org/licenses/>.
+
+import pooch
+from pyxem.data._registry import _file_names_hash, _urls
+import hyperspy.api as hs
+from pathlib import Path
+from pyxem.release_info import version
+from typing import Optional, Union
+import os
+
+# Create a downloader object
+marshall = pooch.create(
```

Comment:
Marshall was the name of my sister's dog. You're free to use it. However, this is a great opportunity to name this particular pooch after a dog in *your* life!

Reply:
:) I like that!